Dockerfile

```diff
@@ -5,5 +5,5 @@ RUN wget https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v
     mv /app/build/bin/grpc_health_probe-linux-amd64 /app/build/bin/grpc_health_probe && \
     chmod +x /app/build/bin/grpc_health_probe
 
-FROM thorax/erigon:stable
+FROM thorax/erigon:2.32.0
```

Comment:
FYI there's `2.32.1` already

Reply:
We do not yet have the corresponding release for `2.32.1`